src/components/molecules/FAQItem/index.tsx

```diff
@@ -11,7 +11,7 @@ const FAQItem: React.FC<FAQItemInterface> = (
 ) => {
     const [open, setOpen] = useState(false);
     return (
-        <StyleFAQItemWrapper onClick={() => setOpen(!open)}>
+        <StyleFAQItemWrapper onClick={() => setOpen(!open)} open={open}>
```

Comment:
Same thing that I told you before, this needs to be something like:

`setOpen(oldProps => !oldProps)`

(I suggest you to ask what's the difference between the two approaches on StackOverFlow or Reactiflux)

Reply:
Oh yeah, thanks! 
I'll try asking about it.